transaction-view/benches/transaction_view.rs

```diff
@@ -64,7 +64,8 @@ fn bench_transactions_parsing(
     group.bench_function("TransactionView (Sanitized)", |c| {
         c.iter(|| {
             for bytes in serialized_transactions.iter() {
-                let _ = TransactionView::try_new_sanitized(black_box(bytes.as_ref())).unwrap();
+                let _ =
```

Comment:
can't comment, but I think there's a benchmark case in here that has >64 txs. We should probably change it, otherwise they will fail.